src/views/wiki/edition/utils/MapInputRow.vue

```diff
@@ -24,16 +24,22 @@
           show-recenter-on
         />
       </div>
-      <div class="column is-6">
+      <div class="column is-5">
         <div class="field">
           <label class="label">{{ $gettext('Longitude') }}</label>
-          <input-simple type="number" postfix="°E" v-model="longitude" @input="setGeometryPoint" />
+          <input-simple type="number" step="any" postfix="°E" v-model="longitude" @input="setGeometryPoint" />
```

Comment:
In case of outings, we should make this inputs read only

Reply:
So the "Pick" button shouldn't be displayed in that case, right ?